LuaMenu/widgets/gui_battle_room_window.lua

```diff
@@ -1291,25 +1291,25 @@ local function InitializeSetupPage(mainWindow, pageConfig, nextPage, selectedOpt
 	}
 	nextButton:Hide()
 
+	advButton = Button:New {

```

Comment:
I think (complete RTS) newbies probably won't click on it in their first game, it's already in the corner and pretty small. Very soon they might get annoyed to have to go to the wizard each time, and they would expect to have the option here rather than inside the setting. 

In general I think it's not good to hide dialog related choices in the settings menu, like we do for a couple of other things (login server settings). Humans are contextual creatures and it feels more intuitive to search for something close to the context in which it's being used/presented. Providing powerful controls and a clean interface is not a zero sum game. Less important things can be hidden in smaller buttons, clickable text links, icons or ultimately under an "advanced" panel. I feel we are greatly under-utilizing icons.

If you are worried about them clicking on the Advanced and feeling confused, not knowing what to do next, I would suggest to consider the following: make such actions easily undoable. In the context of this wizard, it would mean to make the Advanced button togglable and visible in the battleroom window, so they can easily switch back to the wizard if they feel they really don't need all this advanced stuff.

PS: I thought that the skirmish wizard is incomplete? There are a couple of other things that could be improved with it, such as progressing through the wizard by just clicking on the options, without having to click on the "Next" thing as well.